src/controllers/clip.controller.ts

```diff
@@ -38,6 +39,15 @@ const getClip = async (
 ): Promise<Response<any, Record<string, any>>> => {
   const { id } = req.params;
 
+  // checks if the id is a mongodb ObjectID type, otherwise the console spits out an error and won't let you send anymore requests.
+  const ObjectID = Types.ObjectId;
+  const isValidId: boolean = ObjectID.isValid(id);
+  if (isValidId === false) {
```

Comment:
Any reason it isn't like this?
```suggestion
  if (ObjectID.isValid(id) === false) {
```